### ISSUE TITLE
How to encode a single value in CIS JSON?

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/coverage-implementation-schema#24]
If we reduce e.g. a 2D coverage to a single point value (e.g. requesting an elevation value for a specific position), can this be encoded in CIS JSON? Does it need to be encoded as a 1D coverage?

From the ISO 19123 / AT 6 standpoint is there such a thing as a 0D coverage with a single set of range values?

**Comment by pebau** [comment]
extracting a value in the first place would be a value, so the server would return this value (probably in ASCII).

However, for reasons of closure a value might just as well be considered a 0D coverage - so with no axes, an empty axis list, no coordinates, etc. Could certainly be encoded in JSON, for example. (Note that not all formats need to support all result dimensions - eg, TIFF cannot return 5D -  so this is nota "bug".)

Bottom line: behavior in this corner case is implementation dependent, as different applications may expect different things.

**Comment by KathiSchleidt** [comment]
Coming from the Observations&Measurements perspective, my first reaction would be say that this is an observation ;)

Staying within the coverage philosophy where the core data model consists of a Domain and a Range (with some details in between providing context), I'd say the result needs to be a bit more than just an ASCII number (the Range), as without something like the Domain providing the where (in space, time or other conceptual axis), it's really just a number.

I guess one could reduce the Domain to just the origin, then the 0D logic pans out

**Comment by pebau** [comment]
hm, depends:
-  if you did trimming then indeed the result is a coverage, with all its axes, that contains a (single) location, or to be exact: lower bound = upper bound.
- if you arrived via slicing then there is no spoon - err - coordinate any longer as all axes got removed. Meaning, if you keep a coverage view: zero axes in the domain set.

**Comment by pebau** [comment]
coming back to the original question: a coverage that contains only 1 direct position will contain 1 range value, regardless of any regular/irregular/etc bells and whistles, and for any number of coverage dimensions. 
So IMO can be closed.
